downstream/modules/platform/con-gw-understanding-authenticator-mapping.adoc

```diff
@@ -0,0 +1,95 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="gw-understanding-authenticator-mapping"]
+
+= Understanding authenticator mapping
+
+Authentication:: validates a user’s identity, typically through a username and password or a trust system.
+
+Authorization:: determines what an authenticated user can do once they are authenticated. 
+
+In {PlatformNameShort}, authenticators manage authentication, validating users and returning details such as their username, first name, email, and group memberships (for example, LDAP groups). Authorization comes from the authenticator’s associated maps.
+
+During the authentication process, once a user has authenticated, the authorization system starts with a default set of permissions in memory. Then sequentially, the authenticator maps are processed and adjust permissions based on their trigger conditions. Once all the authenticator's maps are processed, the in-memory representation of the users permissions are reconciled with the users existing permissions. 
+
+For example, let's simplify the in-memory representation of the default permissions as follows:
+
+-----
+Access allowed = True
+Superuser permission = Undefined
+Admin of teams = None
+-----
+And, let’s say the maps which need to be processed are processed in the following order:
```

Comment:
Similar to above, removing the first person style: 

```suggestion
And, you might have maps that need to be processed in the following order:
```

Reply:
Fixed.